server/routes/gcp/projects.py

```
@@ -100,136 +97,158 @@ def get_projects(user_id):
+        if not pid:
+            continue
+        result.append(_check_project_iam(
+            crm_service, pid, proj.get('name', pid), member_sa, root_project,
+        ))
+    result.sort(key=lambda x: x['name'])
+    return result
+
+
+@gcp_projects_bp.route("/api/gcp/sa-project-access", methods=["GET", "OPTIONS"])
+@require_permission("connectors", "read")
+def sa_project_access_get(user_id):
```

Ruff (0.15.11)

[warning] Too many return statements (7 > 6)

(PLR0911)